docs/docs/how-to/create_project.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Creating a New Project
+sidebar_position: 90
+---
+
+When you log in to Minder without a project, Minder will automatically create a
+new project to manage your entities (repositories, artifacts, etc). It is also
+possible to create additional projects after you have created your Minder
+profile, for example, to create different projects for different organizations
+or teams you are a part of. Note that
+[Minder Projects](../understand/projects.md) can collect resources from several
+upstream resource providers such as different GitHub organizations, so you can
+register several [entity providers](../understand/providers.md) within a
+project.
+
+## Prerequisites
+
+- A Stacklok account
```

Comment:
I noticed in a previous doc that you switched from Stacklok to a "Minder account", let's homologize on minder accounts.
```suggestion
- A Minder account
```